src/App.tsx

```diff
@@ -21,6 +33,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={perPage}
+            onChange={(e) => {
+              setPerPage(Number(e.target.value));
+              setCurrentPage(1);
+            }}
```

Comment:
it's a good practice to create a separate function instead of passing anonymous function

Reply:
done